src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -1077,6 +1077,22 @@ def get_az_user_agent():
     return ' '.join(agents)
 
 
+def get_az_rest_user_agent():
+    """Get User-Agent for az rest calls"""
+
+    from msrest import __version__ as msrest_version
+    from msrestazure import __version__ as msrestazure_version
+
+    agents = ['python/{}'.format(platform.python_version()),
```

Comment:
is there any existing common util for this?

Reply:
I am searching for that, too. Will replace it if I could find the utils.

Reply:
There should be a similar function in the python sdk, not in azure cli or azure cli core. Just curious, I thought we would remove the dependency of msrest and msrestazure in the future. Is it necessary to add msrest and msrestazure?

Reply:
I agree, removed.